Gemfile.lock

```diff
@@ -1268,8 +1255,6 @@ DEPENDENCIES
   shoulda-matchers
   shrine
   sidekiq (~> 7.2.0)
-  sidekiq-ent!
```

Comment:
Make sure you don't merge any changes to Sidekiq enterprise/pro (see [here](https://depo-platform-documentation.scrollhelp.site/developer-docs/base-setup-vets-api),  step 3).